recipes/pybind11-rdp/meta.yaml

```diff
@@ -0,0 +1,48 @@
+package:
+  name: pybind11-rdp
+  version: 0.1.1
+
+source:
+  url: https://files.pythonhosted.org/packages/57/f3/ba8605feae325db6d493ba604d5a07e08ff3092e83024799966986dc0c23/pybind11_rdp-0.1.1.tar.gz
```

Comment:
Please use the release URL as shown in our example recipe.

https://github.com/conda-forge/staged-recipes/blob/43cb020eb60984df338fd750d7a6ba1ecf3c4257/recipes/example/meta.yaml#L16